app/services/bill-runs/two-part-tariff/allocate-returns-to-charge-element.service.js

```diff
@@ -10,43 +10,32 @@ const { periodsOverlap } = require('../../../lib/general.lib.js')
 /**
  * For a chargeElement with matching returns any abstracted volume recorded on the return log will be allocated to the
  * charge element up to a maximum of the charge elements authorised volume, or the remaining authorised volume on the
- * charge reference, whichever is lower.
+ * charge reference, whichever is lower. If any "due" returns are are matched to a charge element, that element will
```

Comment:
Just a typo (great explanation BTW)

```suggestion
 * charge reference, whichever is lower. If any "due" returns are matched to a charge element, that element will
```

Reply:
Thanks. I'll sort the typo.